bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -77,6 +77,14 @@ const (
 // The namespace for Istio
 const IstioNamespace = "istio-system"
 
+var DEFAULT_EXP_BACKOFF = backoff.ExponentialBackOff{
+	InitialInterval:     backoff.DefaultInitialInterval,
```

Comment:
Do you want to make InitialInterval bigger?

Reply:
done.